pkgs/tools/misc/doitlive/default.nix

```
@@ -2,11 +2,11 @@
 
 python3Packages.buildPythonApplication rec {
   pname = "doitlive";
-  version = "4.3.0";
+  version = "5.0.0";
```

```suggestion
  version = "5.0.0";
  pyproject = true;
```
and add `setuptools` to `nativeBuildInputs`.